Makefile

```diff
@@ -16,7 +17,7 @@ ifeq ($(LINT_OS),Darwin)
 endif
 
 LINT_LOWER_OS  = $(shell echo $(LINT_OS) | tr '[:upper:]' '[:lower:]')
-GOLINT_CONFIG:=$(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))/.golangci.yml
+GOLINT_CONFIG:=$(shell dirname $(LINT_ROOT)/.golangci.yml)
```

Comment:
isn't this just `$(LINT_ROOT)`?

Reply:
It points to a file within $LINT_ROOT -- I'm not sure what you are asking?

Reply:
I get it - the Makefile had not been updated.